1750534004141	Sync.LogManager	DEBUG	Flushing file log
1750534004143	Sync.LogManager	DEBUG	Log cleanup threshold time: 1749670004142
1750534004144	Sync.LogManager	DEBUG	Done deleting files.
1750534005213	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1750537175788	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1750537175788	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1750537175789	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1750537175792	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1750537178800	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1750537178800	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1750537178802	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1750537178804	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1750537178805	Sync.Service	DEBUG	User-Agent: Zen/1.13.2b (Linux x86_64) FxSync/1.141.0.20250616094514.desktop
1750537178805	Sync.Service	INFO	Starting sync at 2025-06-21 22:19:38 in browser session Aj5SXGzIFQhx
1750537178805	Sync.Service	DEBUG	In sync: should login.
1750537178805	Sync.Service	INFO	User logged in successfully - verifying login.
1750537178805	Sync.SyncAuthManager	DEBUG	unlockAndVerifyAuthState has no FxA user
1750537178805	Sync.Status	DEBUG	Status.login: error.login.reason.no_username => error.login.reason.no_username
1750537178805	Sync.Status	DEBUG	Status.service: service.client_not_configured => service.client_not_configured
1750537178805	Sync.Service	DEBUG	Fetching unlocked auth state returned error.login.reason.no_username
1750537178805	Sync.ErrorHandler	ERROR	Sync encountered a login error
1750537178805	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1750537178805	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1750537178806	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.no_username(resource://services-sync/service.sys.mjs:1051:15) JS Stack trace: onNotify@service.sys.mjs:1051:15
1750537178806	Sync.Service	DEBUG	Not syncing: login returned false.
